code/modules/projectiles/projectile.dm

```diff
@@ -348,3 +348,8 @@
 
 /obj/item/projectile/experience_pressure_difference()
 	return
+
+
+/obj/item/projectile/proc/dumbfire(var/dir)
+	current = get_ranged_target_turf(src, dir, world.maxx) //world.maxx is the range. Not sure how to handle this better.

```

Comment:
```world.maxx``` -> ```max(world.maxx, world.maxy)``` that way it will go as far as the longest edge (SS13 maps are typically square but hey)